reactor-core/src/main/java/reactor/core/scheduler/Schedulers.java

```diff
@@ -1000,6 +1008,29 @@ default Scheduler newBoundedElastic(int threadCap, int queuedTaskCap, ThreadFact
 			return new BoundedElasticScheduler(threadCap, queuedTaskCap, threadFactory, ttlSeconds);
 		}
 
+		/**
+		 * {@link Scheduler} that dynamically creates a bounded number of
+		 * Workers, reusing them once the Workers have been shut down. The underlying (user or daemon)
+		 * threads can be evicted if idle for more than {@code ttlSeconds}.
+		 * <p>
+		 * The maximum number of created thread pools is bounded by the provided {@code cap}.
+		 * <p>
+		 * The main difference between {@link BoundedElasticScheduler} and
+		 * {@link LoomBoundedElasticScheduler} is that underlying workers allocates a
+		 * new thread for every new task which is the main requirements for
+		 * {@link VirtualThread}s
+		 *
+		 * @param threadCap maximum number of underlying threads to create
```

Comment:
I'd go with a different wording: "maximum number of simultaneously executing VirtualThreads"

Reply:
We don't have the direct binding to VirtualThreads here although the name suggests Loom

Reply:
renamed class